File: easy/raindrops.py
def raindrops(drop):
    """
    DOCUMENTATION
    """
    bank = [
        "Pling",
        "Plang",
        "Plong",
    ]
    target = ""
    for index, num in enumerate([3, 5, 7]):
        if drop % num == 0:
            target += bank[index]

    return target if len(target) > 0 else str(drop)

File: src/easy/raindrops.ts
export const raindrops = (drop: number): string => {
    const modulus = [3, 5, 7];
    const target = ["Pling", "Plang", "Plong"].reduce(
        (a: string, c: string, i: number) =>
            drop % modulus[i] !== 0 ? a : a + c,
        "",
    );
    return target.length > 0 ? target : `${drop}`;
};
